### ISSUE TITLE
read-only not announced for checkboxes

### ISSUE THREAD

**Issue by kanggeon** [nvaccess/nvda#6563]
Read only state is not announced for checkboxes with aria-readonly set. The latest [ARIA specifications](http://www.w3.org/TR/wai-aria-1.1/#aria-readonly) include checkbox as an aria-readonly compatible role. It is possible for sighted users to know the read-only state without interacting with the checkbox through different styling of "read-only" checkboxes. This is not possible for NVDA users.

**Comment by feerrenrut** [comment]
I imagine this would be a fairly quick fix, and provides good value. I would be interested in some example of this in the wild? Any comment on how often this crops up?
Assigning this a priority 2, based on being quick to implement.
